src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/EventsController.cs

```diff
@@ -126,6 +151,12 @@ public async Task<ActionResult<string>> Post([FromBody] CloudEvent cloudEvent)
             {
                 List<CloudEvent> events = await _eventsService.Get(after, from, to, party, source, type, size);
 
+                if (events.Count > 0)
+                {
+                    // Authorizes the events.
```

Comment:
The comment doesn't really add any extra information here, could be removed.

Reply:
The same as my girfriend tell me when i open my mouth. Removed. :-)